packages/arb-token-bridge-ui/src/components/common/AddCustomChain.tsx

```diff
@@ -177,6 +177,7 @@ async function mapOrbitConfigToOrbitChain(
     (await rollup.confirmPeriodBlocks()).toNumber() ?? 150
   return {
     chainID: data.chainInfo.chainId,
+    slug: '',
```

Comment:
what's this one for? wouldn't we want to accept a slug and fallback to empty?

Reply:
Slug is required for Orbit chains added to the config. Custom Orbit chains don't use it for anything. In query params we use slug for Orbit chains from config, and chain ID for custom chains

Reply:
https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1429/commits/a8f4c86f1351e711a3e21ad09913dcb6aa1712b3

added a comment